From caaec484d88fe4dfc1f83b8ce2e2ff1167193517 Mon Sep 17 00:00:00 2001
From: Yunfeng Ye <yeyunfeng@huawei.com>
Date: Thu, 2 Jul 2020 10:22:21 +0800
Subject: [PATCH] rt-tests: cyclictest: remove the debug log "pid = xxx" in
 rstat_shm_open()

The infomation "pid = xxx" is printed with other infomation on the same
line. so remove this left over debug message in rstat_shm_open().

Signed-off-by: yeyunfeng <yeyunfeng@huawei.com>

Edited the descriptioin in the log a little bit.
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 src/cyclictest/cyclictest.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/cyclictest/cyclictest.c b/src/cyclictest/cyclictest.c
index 79bb1cb71c68..eb61be394f2e 100644
--- a/src/cyclictest/cyclictest.c
+++ b/src/cyclictest/cyclictest.c
@@ -1921,7 +1921,6 @@ static int rstat_shm_open(void)
 	pid_t pid;
 
 	pid = getpid();
-	printf("pid = %d", pid);
 
 	snprintf(shm_name, SHM_BUF_SIZE, "%s%d", "/cyclictest", pid);
 
-- 
2.21.3

